src/inc/msquic.h

```diff
@@ -1371,6 +1374,9 @@ typedef enum QUIC_STREAM_EVENT_TYPE {
     QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE         = 7,
     QUIC_STREAM_EVENT_IDEAL_SEND_BUFFER_SIZE    = 8,
     QUIC_STREAM_EVENT_PEER_ACCEPTED             = 9,
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
+    QUIC_STREAM_EVENT_PEER_RELIABLE_ABORT_SEND  = 10,
+#endif
```

Comment:
You don't actually need to create a new event. Just use the normal abort after you've reliably delivered the necessary bytes.